raiden-ts/package.json

```diff
@@ -9,13 +9,14 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "start": "npm run contracts && tsc -p src/tsconfig.cjs.json -w",
-    "build": "npm run contracts && tsc -p src/tsconfig.json && tsc -p src/tsconfig.cjs.json",
+    "build": "npm run contracts && tsc -p src/tsconfig.json && tsc -p src/tsconfig.cjs.json && npm run contracts:post",
     "test": "npm run lint && NODE_ENV=development jest --coverage=true",
     "lint": "eslint src/**/*.ts",
-    "clean": "rimraf dist/ dist:cjs/ .coverage/ contracts/ src/abi/ src/deployment/",
-    "contracts:copy": "node ./scripts/copyContracts.js",
-    "contracts:build": "typechain --target=ethers 'src/abi/*.json' --outDir contracts",
-    "contracts": "npm run contracts:copy && npm run contracts:build",
+    "clean": "rimraf dist/ dist:cjs/ .coverage/ src/contracts/ src/abi/ src/deployment/",
+    "contracts:pre": "node ./scripts/copyContracts.js prebuild",
```

Comment:
`prebuild` is unecessary because we don't check for that specific argument

Reply:
I know. But as it's also accepted, it's here as "documentation", it's clear when it's run. The fact that we don't check it is just an implementation detail, and we could do.

Reply:
Fixed by making it required in `copyContracts.js`